Equilobe.DailyReport.SL/TaskSchedulerService.cs

```diff
@@ -71,7 +72,7 @@ public void CreateOrUpdateTask(string taskKey, string toolPath, string arguments
                 if (trigger != null)
                     taskDefinition.Triggers.Add(trigger);
 
-                GetTaskFolder(ts).RegisterTaskDefinition(taskKey, taskDefinition, TaskCreation.CreateOrUpdate, WindowsIdentity.GetCurrent().Name);
+                //GetTaskFolder(ts).RegisterTaskDefinition(taskKey, taskDefinition, TaskCreation.CreateOrUpdate, WindowsIdentity.GetCurrent().Name);
```

Comment:
Why is this commented? Shouldn't the task be registered?

Reply:
I commented this, and forgot to uncomment, because i don't want the scheduled task to be created when I update the dashboard

Reply:
I will be more careful to avoid having this kind of a thing in a pull request.